labs/python_at_the_edge/1.md

```diff
@@ -15,9 +15,9 @@ Please ensure you have completed Linux at the Edge: Introduction to Guest Shell
 
 ## Cisco Guest Shell Capabilities
 
-Guestshell is a virtualized Linux-based environment,designed to run custom Linux applications,including Python for automated control and management of Cisco devices.Using GuestShell, users can also install update,and operatethird-party Linux applications. Guest Shell containeris managed using IOx. IOx is Cisco's Application Hosting Infrastructure for Cisco IOSXE devices.IOx enables hosting of applications and services developed by Cisco, partners,and third-party developers in network edge devices.
+Guest shell is a virtualized Linux-based environment, designed to run custom Linux applications including Python for automated control and management of Cisco devices. Using guest shell, users can also install, update, and operate third-party Linux applications. The guest shell container is managed using IOx. IOx is Cisco's Application Hosting Infrastructure for Cisco IOS XE devices. IOx enables hosting of applications and services developed by Cisco, partners, and third-party developers in network edge devices.
 
-The below table shows the Guest Shell capabilitiesand the supported Cisco platforms
+The below table shows the guest shell capabilities and the supported Cisco platforms.
 
 ![](/posts/files/python_at_the_edge/images/guest_shell_cap.jpg)
```

Comment:
This is a table you could convert to Markdown I believe. I use this table formatter if it helps you generate one: http://www.tablesgenerator.com/markdown_tables